docs/README.md

```diff
@@ -47,6 +47,15 @@ The following CSI interfaces are implemented:
  * The uid/gid configured on the access point is either the uid/gid specified in the storage class, a value in the gidRangeStart-gidRangeEnd (used as both uid/gid) specified in the storage class, or is a value selected by the driver is no uid/gid or gidRange is specified.
  * We suggest using [static provisioning](https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/examples/kubernetes/static_provisioning/README.md) if you do not wish to use user identity enforcement.
 
+**Note**
+
+If you want to pass any other mountOptions to EFS CSI driver while mounting, they can be passed in pv and storage class for static and dynamic provisioning respectively.
```

Comment:
I would capitalize the Kubernetes resources ->

...they can be passed in through the Persistent Volume or the Storage Class objects, depending on whether static or dynamic provisioning is used.